velox/connectors/hive/HiveConfig.h

```diff
@@ -247,6 +251,10 @@ class HiveConfig {
 
   bool s3UseProxyFromEnv() const;
 
+  /// Returns the timestamp unit used when exporting to Arrow.
+  /// 0: second, 3: milli, 6: micro, 9: nano.
+  uint8_t arrowBridgeTimestampUnit(const Config* session) const;
```

Comment:
Does Arrow perform the error handling for values outside these?
Should Velox be performing a check as well?

Reply:
Yes, Arrow throws exception for unexpected unit during conversion ([link](https://github.com/apache/arrow/blob/main/cpp/src/arrow/c/bridge.cc#L416-L417)). Added a check here, thanks.